src/main/webapp/go.js

```diff
@@ -103,18 +164,28 @@ function createLine(text) {
 
 /**
  * Show or hide the proceed button.
- *
  * @param {boolean} hide Whether the proceed button should be hidden or shown.
  */
 function toggleProceedButton(hide) {
   const proceedButton = document.getElementById(PROCEED_BUTTON);
   if (hide) {
     proceedButton.classList.add(INVISIBLE_CLASS);
   } else {
+    if (destIndex == huntArr.length - 1) {
```

Comment:
You might want to use === instead of == here because it will enforce value and type equality
```suggestion
    if (destIndex === huntArr.length - 1) {
```